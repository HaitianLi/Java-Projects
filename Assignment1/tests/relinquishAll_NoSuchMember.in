add member jack hu
relinquish all 100001
exit