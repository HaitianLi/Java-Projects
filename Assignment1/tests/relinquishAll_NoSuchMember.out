user: Success.

user: No such member in system.

user: Ending Library process.
